.github/workflows/stress-test.yml

```diff
@@ -0,0 +1,35 @@
+# Nightly Stress Test for self-hosted runners
+name: Self-hosted Runners Nightly Stress Test
+on:
+  schedule:
+    # Triggers at 11pm every night.
+    - cron: '0 23 * * *'
```

Comment:
I recommend adding a workflow_dispatch to this as well for testing and manual invocation

Reply:
Added.